html/annotated.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
You should remove html and latex from your git index

Reply:
i see, although i put them in the .gitignore before i even made a first commit. Welp, no helping that